### ISSUE TITLE
[BUG] Enable PHP session.use_strict_mode breaks login

### ISSUE THREAD

**Issue by LufinityLucas** [Leantime/leantime#2558]
**What set up are you using**
* [ ] Cloud Hosted Version
* [x] Self Hosted

**Describe the bug**
PHP encourage all sites to enable the PHP setting `session.use_strict_mode` to prevent session fixation. See the [PHP documentation](https://www.php.net/manual/en/session.configuration.php#ini.session.use-strict-mode). When enabling this in the latest Docker Leantime image (probably in other environments as well), it breaks the login process. Whenever you login, either through OIDC or local user, you are bounced back to the login screen.

**To Reproduce**
In my case, as I'm using Docker, the follwing steps were followed:
1. Create your own Dockerfile and enter e.g.,
```
FROM leantime/leantime:latest
```
2. Make sure to copy the default production PHP ini file, and enable the `session.use_strict_mode` setting using e.g.,
```
RUN mv /usr/local/etc/php/php.ini-production /usr/local/etc/php/php.ini
sed -i "/session.use_strict_mode =/c\session.use_strict_mode = On" /usr/local/etc/php/php.ini
```
3. Build and run the container, and try to login with e.g., a local Leantime account.

**Expected behavior**
Enabling the `session.use_strict_mode` should not break the login function. After providing the right credentials, the user should be taken into the application instead of bounced back to the login page.

**Leantime Version**
3.1.4

**Server**
Docker Desktop on Windows

**Comment by marcelfolaron** [comment]
This should be fixed with https://github.com/Leantime/leantime/pull/2530
I'll keep an eye on it.

**Comment by LufinityLucas** [comment]
> This should be fixed with #2530 I'll keep an eye on it.

That's what i suspected as well. Thanks! 😄

**Comment by marcelfolaron** [comment]
With 3.2.0 out this is now fixed
